paywall/src/utils/postOffice.js

```diff
@@ -34,10 +35,28 @@ export function setupPostOffice(window, target, targetOrigin) {
     const listeners = handlers[event.data.type]
     if (listeners && listeners.size) {
       listeners.forEach(listener => {
+        if (debug) {
```

Comment:
Do you think these could be factored into a `debug()` function that just wraps the `if (debug)` and `console.log` parts?

Reply:
I do. However, this will have a performance impact even when debug is off, as the function will always be called, and not eliminated by dead code removal, so I elected not to do this

Reply:
Fair enough